doc/changes/makedocreldata.g

```diff
@@ -9,7 +9,8 @@ GAPInfo.ManualDataChanges:= rec(
   bookname:= "changes",
   pathtoroot:= "../..",
 
-  files:= [ ],
+  files:= [
```

Comment:
I guess this change is here to use the same style as in other `doc/*/makedocreldata.g` files, so that in the future any files containing documentation sections will be added between these two lines.